docs/src/index.md

```diff
@@ -34,8 +34,15 @@ tool for working with tabular data in Julia -- as noted below, there are some
 other great libraries for certain use-cases -- but it provides great data
 wrangling functionality through a familiar interface.
 
-To understand the toolchain in more detail, have a look at the tutorials in this manual. New
-users can start with the [First Steps with DataFrames.jl](@ref) section.
+To understand the toolchain in more detail, have a look at the tutorials in this manual.
+New users can start with the [First Steps with DataFrames.jl](@ref) section.
+
+If you do not have a significant programming experience you might find 
+[DataFramesMeta.jl](https://juliadata.github.io/DataFramesMeta.jl/stable/) package
+or one of the other convenience packages discussed in
+[Data manipulation frameworks](@ref) section of this manual
+helpful when writing more advanced data transformations. These packages provide
+convenience syntax similar to [dplyr](https://dplyr.tidyverse.org/) in R.
```

Comment:
Maybe mention programming experience at the end, as these macros are convenient even if you're an experienced programmer?
```suggestion
You may find the [DataFramesMeta.jl](https://juliadata.github.io/DataFramesMeta.jl/stable/)
package or one of the other convenience packages discussed in
the [Data manipulation frameworks](@ref) section of this manual
helpful when writing more advanced data transformations,
especially if you do not have a significant programming experience.
These packages provide convenience syntax similar to [dplyr](https://dplyr.tidyverse.org/) in R.
```